tabs/osd.js

```diff
@@ -246,12 +247,14 @@ OSD.constants = {
   ALL_DISPLAY_FIELDS: {
     MAIN_BATT_VOLTAGE: {
       name: 'MAIN_BATT_VOLTAGE',
+      desc: 'Instantaneous main battery voltage (flashes when low)',
```

Comment:
Can you please extract the strings into https://github.com/betaflight/betaflight-configurator/blob/master/_locales/en/messages.json , like it is done in the rest of Betaflight configurator?